xline/src/storage/kv_store.rs

```diff
@@ -147,6 +170,16 @@ where
         self.revision.get()
     }
 
+    /// Get compacted revision of  KV store
+    pub(crate) fn compacted_revision(&self) -> i64 {
+        self.compacted_rev.load(Relaxed)
```

Comment:
I recommend to use `Release` here. The ordering of revision load and update is important.